### PR TITLE
Lots of UX changes, defaulting to pick up env variables, and normalizing some test stuff

### PR DESCRIPTION
**Pull request by aronchick**

### CODE REVIEW

File: cmd/bacalhau/docker_run.go [hunk 1]
```diff
@@ -103,38 +123,20 @@ var dockerRunCmd = &cobra.Command{
...
-/bin/zsh
-/usr/bin/zsh`, "/n")
-
-		containsGlob := false
-		for _, entrypointArg := range jobEntrypoint {
-			if strings.ContainsAny(entrypointArg, "*") {
-				containsGlob = true
-			}
+		for _, i := range jobInputs {
+			jobInputVolumes = append(jobInputVolumes, fmt.Sprintf("%s:/inputs", i))
```
Comment (lukemarsden) [inline cmd/bacalhau/docker_run.go:127]: If user specifies this multiple times will it clobber /inputs mountpoint and error?

File: cmd/bacalhau/docker_run.go [hunk 2]
```diff
@@ -103,38 +123,20 @@ var dockerRunCmd = &cobra.Command{
...
-				if strings.Index(strings.TrimSpace(jobEntrypoint[0]), shell) == 0 {
-					containsGlob = false
-					break
-				}
-			}
-			if containsGlob {
-				log.Warn().Msgf("We could not help but notice your command contains a glob, but does not start with a shell. This is almost certainly not going to work. To use globs, you must start your command with a shell (e.g. /bin/bash <your command>).") // nolint:lll // error message
-			}
+		// No error checking, because it will never be an error (for now)
+		sanitizationMsgs, sanitzationFatal := system.SanitizeImageAndEntrypoint(jobEntrypoint)
```
Comment (lukemarsden) [inline cmd/bacalhau/docker_run.go:131]: Typo in 2nd arg

File: cmd/bacalhau/docker_run_test.go [hunk 3]
```diff
@@ -63,15 +69,177 @@ func (suite *DockerRunSuite) TestRun_GenericSubmit() {
...
+			err           error
+		}{
+			{outputVolumes: []OutputVolumes{{name: "", path: ""}}, err: nil}, // Flag not provided
+		}
+
+		_ = i
+		_ = tc
+		_ = testCids
+
+		// for _, tcids := range testCids {
```
Comment (lukemarsden) [inline cmd/bacalhau/docker_run_test.go:191]: Why commented out?

**Review by lukemarsden — APPROVED** [review]
This looks great thanks! Feel free to merge and address any issues raised as follow-up, or fix and merge, your call 👍

**Comment by lukemarsden** [comment]
Do we only allow one -i?

File: cmd/bacalhau/docker_run_test.go (hunk 3 above)
Comment (aronchick) [reply cmd/bacalhau/docker_run_test.go:191]: checked in wrong version - changing

**Comment by lukemarsden** [comment]
Cool, merge at will and please explain how multiple -i's get mounted at the
same time :-)

**Comment by aronchick** [comment]
> 



> Cool, merge at will and please explain how multiple -i's get mounted at the same time :-)

OH! No, i don't check on that (and should we? or fail?) I guess, my thinking was that if you're writing the same mount point for both, we'd just mount them both into that folder, but that seems broken.

**Pull request merged**